### ISSUE TITLE
README instructions leading to missing file browser.js

### ISSUE THREAD

**Issue by philip-peterson** [chinedufn/percy#134]
After following the README instructions, I was having trouble getting the desired output to show up. The file `browser.js`, while referenced in `index.js`, seems to be not specified anywhere, leading to a blank page when starting the app.

**Comment by chinedufn** [comment]
Hey @philip-peterson ,

Took me some time to get back to you due to travel - so sorry about that. Thanks a lot for opening this issue.

I just confirmed that the instructions should work.

I suspect that you might have copied the dependencies from the README without copying the `Cargo.toml` entirely.

The `Cargo.toml` package is named `browser` in the README - but yours is probably named `client-side-web-app`

---

I've made things less confusing by naming the package `client-side-web-app` here.

https://github.com/chinedufn/percy/commit/e0026e88b1bebed20eff38fada8f96d6e8ba2cf0

If you try following the instructions again everything should work!
